TombEngine/Specific/level.cpp

```diff
@@ -85,50 +85,50 @@ std::vector<int> StaticObjectsIds;
 std::vector<int> SpriteSequencesIds;
 LEVEL g_Level;
 
-unsigned char ReadUInt8()
+static unsigned char ReadUInt8()
```

Comment:
Why do you introduce `static` keyword for global functions, which are already implcitly static?

Reply:
This goes back to a discussion we had last year about best practice in this regard. It limits the visibility of functions to the specific file in which they are used (reducing the risk of cryptic naming conflicts upon compilation) and helps encapsulate the implementation. IMO I find the best benefit is readability, since it visually indicates when a function is only intended for use within a specific file.